.github/workflows/build-snap.yml

```diff
@@ -125,6 +125,8 @@ jobs:
     steps:
       - name: Checking out repo
         uses: actions/checkout@v3.3.0
+      - name: Setup tmate session
```

Comment:
Do we need this? Maybe we can keep it for debugging sessions

Reply:
It blocks the pipeline, I'll comment it out.